Client/Frontend/Home/BookmarksPanel.swift

```diff
@@ -23,7 +23,7 @@ class BookmarksPanel: SiteTableViewController, HomePanel {
     weak var homePanelDelegate: HomePanelDelegate? = nil
     var source: BookmarksModel?
     var parentFolders = [BookmarkFolder]()
-    var bookmarkFolder = BookmarkRoots.MobileFolderGUID
+    var bookmarkFolder: BookmarkFolder?
```

Comment:
If we never assign `nil` to this — making it non-optional — we could do `= BookmarkFolder(BookmarkRoots.MobileFolderGUID, BookmarksFolderTitleMobile)` here, and skip the rest of the changes in this diff.